lib/config/error-codes/security.json

```diff
@@ -176,6 +176,36 @@
           "message": "Missing or malformed \"test\" attribute  for role %s (%s) : %s",
           "class": "BadRequestError",
           "deprecated": true
+        },
+        "unknown_controller": {
```

Comment:
I find this error ok (but not others, see my comments), because either way, the provided controller is effectively unknown to the role validator.

I would add a didYouMean hint to the message however